src/main/java/org/wikimedia/elasticsearch/swift/repositories/blobstore/SwiftBlobContainer.java

```diff
@@ -136,15 +139,15 @@ public void deleteBlobsIgnoringIfNotExists(List<String> blobNames) throws IOExce
                     }
                 }
             }
-            return blobsBuilder.build();
+            return blobsBuilder.immutableMap();
         });
     }
 
     /**
      * Get all the blobs
      */
     @Override
-    public ImmutableMap<String, BlobMetaData> listBlobs() {
+    public Map<String, BlobMetaData> listBlobs() {
```

Comment:
Maybe separate this change since it's an API change? 
Or was this an Elasticsearch change?

Reply:
It's because the guava deps were dropped. For details see [here](https://github.com/elastic/elasticsearch/issues/13224)